operations/observability/mixins/meta/rules/server.yaml

```diff
@@ -46,7 +46,7 @@ spec:
 
     # Rollout alerts
     - alert: JsonRpcApiErrorRates
-      expr: sum(rate(gitpod_server_api_calls_total{statusCode!~"2.*|3.*|4.*"}[5m])) by (cluster) / sum(rate(gitpod_server_api_calls_total[5m])) by (cluster) > 0.04
+      expr: sum(rate(gitpod_server_api_calls_total{statusCode!~"2.*|3.*|4.*|640"}[5m])) by (cluster) / sum(rate(gitpod_server_api_calls_total[5m])) by (cluster) > 0.04
```

Comment:
What if we instead used standard 4xx error code to represent this case instead of using custom ones? It would ensure we don't need to exclude/include every special code in the metrics

Reply:
We should, yes. For now, let's exclude to take the burden off L1.

Reply:
We can silence the alert for that particular error code which avoids needing this change, but up to you. In any case, follow-up would be good

Reply:
I have it silenced through today, but there's probably value in this alert surfacing 5xx errors, so didn't want to silence it for too long. I agree though that a 4xx would be better here. We have a handful of 6xx codes it seems like, so maybe a task to replace those would be prudent as a followup.